### PR TITLE
Update prettier version

### PR DESCRIPTION
**Pull request by DanielHabenicht**

### CODE REVIEW

File: prettier/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM node:6.9.1-slim
```
Comment (DanielHabenicht) [inline prettier/Dockerfile:1]: maybe also update the node version? I haven't tested it.

Comment (pbrisbin) [reply prettier/Dockerfile:1]: No expertise or opinions here from me. Happy to defer to you to try that out. If the test suite stays green I'd be fine with it.

File: prettier/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 MAINTAINER Pat Brisbin <pbrisbin@gmail.com>
 ENV LANG en_US.UTF-8
-ENV PRETTIER_VERSION 1.19.1
+ENV PRETTIER_VERSION 2.0.2
```
Comment (pbrisbin) [inline prettier/Dockerfile:4]: You'll want to update the `info.yaml` as well. We tag the images to convey the underlying tools version.

**Comment by pbrisbin** [comment]
It also looks like something's up in this version:

```
/usr/local/lib/node_modules/prettier/bin-prettier.js:6702
  return (x[0] - y[0]) ** 2 + (x[1] - y[1]) ** 2 + (x[2] - y[2]) ** 2;
                        ^
SyntaxError: Unexpected token *
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

Any ideas about that?

File: prettier/Dockerfile (hunk 2 above)
Comment (DanielHabenicht) [reply prettier/Dockerfile:4]: I updated it.

**Comment by DanielHabenicht** [comment]
> It also looks like something's up in this version:
> 
> ```
> /usr/local/lib/node_modules/prettier/bin-prettier.js:6702
>   return (x[0] - y[0]) ** 2 + (x[1] - y[1]) ** 2 + (x[2] - y[2]) ** 2;
>                         ^
> SyntaxError: Unexpected token *
>     at Object.exports.runInThisContext (vm.js:76:16)
>     at Module._compile (module.js:542:28)
>     at Object.Module._extensions..js (module.js:579:10)
>     at Module.load (module.js:487:32)
>     at tryModuleLoad (module.js:446:12)
>     at Function.Module._load (module.js:438:3)
>     at Module.runMain (module.js:604:10)
>     at run (bootstrap_node.js:394:7)
>     at startup (bootstrap_node.js:149:9)
>     at bootstrap_node.js:509:3
> ```
> 
> Any ideas about that?

Yep, that was indeed the node version, that was too old. I updated it. It should be running fine now. 
Just for your info, Prettier 2.0 had some breaking changes regarding default values for formats, these will be applied to all projects after this gets merged.

File: prettier/Dockerfile (hunk 1 above)
Comment (DanielHabenicht) [reply prettier/Dockerfile]: updated it to latest LTS version

**Comment by pbrisbin** [comment]
> will be applied to all projects after this gets merged

A few caveats here:

- Nothing happens to projects using default configuration until I release this as a new [Restylers Version](https://github.com/restyled-io/restyled.io/wiki/Restyler-Versions#how-it-works), and update the [default](https://github.com/restyled-io/restyler/blob/master/config/default.yaml#L110) to be it
- Any users for whom the Prettier version matters, can [explicitly specify the `image`](https://github.com/restyled-io/restyled.io/wiki/Restyler-Versions#avoid-specific-surprises) in their config

Once I merge this (and push up the Docker image), anyone will be able to use it in their own `image` configuration though.

**Pull request merged**